tests/test_planner/rrt_test.py

```diff
@@ -14,16 +15,25 @@ def test_rrt():
         ],
     ]
 
+    poly = PolygonEnv(Environment)
+    # poly.update(general_obstacles_list)
+
     for obstacles in general_obstacles_list:
-        obstacle_list = obstacles
+        poly.update(obstacles)
 
         # Instatiate rrt planner object
         my_tree = RRT(sample_area=(-5, 15), sampler=sampler, expand_dis=0.1)
-        path, node_list = my_tree.plan((1, 1), (10, 10), obstacle_list)
+        path, node_list = my_tree.plan((1, 1), (10, 10), poly)
 
         # RRT.visualize_tree(node_list, obstacle_list)
-        optimized_path = path_optimizer(path, obstacle_list)
-        # from gennav.utils.planner import visualize_path
-        # visualize_path(optimized_path, obstacle_list)
+        
+        # Convert path (list of Nodes) to Tajectory (list of RobotStates) 
+        for node in path:
+            node = RobotState(position=node.state.position)
+        path = Trajectory(path)
+        
+        optimized_path = path_optimizer(path, poly)
```

Comment:
No need to call this in rrt_test, it can be tested separately

Reply:
Cool, I'll _remove_ the line then. It was there in the former rrt_test.